### ISSUE TITLE
EAL: Unable to open runtime directory

### ISSUE THREAD

**Issue by liang2yan** [spdk/spdk#790]
## Expected Behavior
SPDK nvme_manage can run without error.

## Current Behavior
nvme_manage reported following error.
```
Starting SPDK v19.07-pre / DPDK 18.11.1 initialization...
[ DPDK EAL parameters: nvme_manage --no-shconf -c 0x1 --log-level=lib.eal:6 --log-level=user1:6 --base-virtaddr=0x200000000000 --file-prefix=spdk_pid155418 ]
EAL: No free hugepages reported in hugepages-1048576kB
EAL: Unable to open runtime directory
EAL: Error while clearing runtime dir: No such file or directory
EAL: FATAL: Cannot clear runtime directory

EAL: Cannot clear runtime directory

Failed to initialize DPDK
Unable to initialize SPDK env

```
## Steps to Reproduce
1. get latest SPDK codes.
2. configure SPDK with following command:

`./configure --without-isal --without-ocf --with-dpdk=/pathtodpdk-stable-18.11.1/x86_64-native-linuxapp-gcc`

3. run "make" command to compile spdk.
4. run spdk/examples/nvme_manage/nvme_manage command.
`./examples/nvme/nvme_manage/nvme_manage
`
## Context (Environment including OS version, SPDK version, etc.)
This issue exists with latest SPDK and SPDK-19.04.

**Comment by liang2yan** [comment]
@darsto , should we add this test case into the jenkins test pool?

**Comment by darsto** [comment]
@liang2yan The failure has nothing to do with SPDK. DPDK is complaining:
> EAL: Unable to open runtime directory

That means you don't have access to neither /var/run/dpdk nor /tmp/dpdk

Are you running with sudo?

**Comment by liang2yan** [comment]
Yes, I run with root user account. I have run SPDK with DPDK 18.11 before. is it conflict with DPDK18.11.1?

**Comment by darsto** [comment]
Hmm, I'll try to reproduce this

**Comment by GaoliangL** [comment]
It’s a DPDK 18.11.1 issue, not a defect in SPDK 19.04.
DPDK 18.11.1 release included the backported patch ‘3d8191d eal: clean up unused files on initialization’ , but missed its follow-up fix ‘4c1ef6d eal: fix cleanup in no-shconf mode’. It will cause the failure ‘EAL: Cannot clear runtime directory’ when starting any APPs with the EAL option ‘--no-shconf’.

**Comment by changpe1** [comment]
@GaoliangL Thanks for pointing that out, @liang2yan Can you try with a new DPDK version, we can close this issue if it's not related with SPDK.

**Comment by liang2yan** [comment]
Tried SPDK-19.04 with DPDK-18.11 and DPDK-19.02. No error reported. So I think we can close this issue.

**Comment by darsto** [comment]
FYI, this is fixed in the upcoming DPDK 18.11.2
